app/models/application.rb

```diff
@@ -10,9 +10,9 @@ def find_all_countries
       end
     end
 
-    def find_country_by_iso(iso)
-      return nil unless iso
-      url = "#{ENV['CARTO_API']}/sql?q=SELECT%20iso%2C%20name_engli%20as%20name%20FROM%20gadm36_countries%20WHERE%20iso%20%3D%20%27#{iso}%27%20AND%20iso%20!%3D%20%27XCA%27%20AND%20iso%20!%3D%20%27TWN%27"
+    def find_adm0_by_adm0_id(adm0)
+      return nil unless adm0
+      url = "#{ENV['CARTO_API']}/sql?q=SELECT%20iso%2C%20name_engli%20as%20name%20FROM%20gadm36_countries%20WHERE%20iso%20%3D%20%27#{adm0}%27%20AND%20iso%20!%3D%20%27XCA%27%20AND%20iso%20!%3D%20%27TWN%27"
```

Comment:
if I can barge in, this could be made more readable by using the following format:

````
url = "#{ENV['CARTO_API']}/sql?q="
url += <<-SQL
  SELECT iso, name_engli as name
  FROM gadm36_countries
  WHERE iso = '#{adm0}' AND iso != 'XCA' AND iso != 'TWN'
SQL
````

Should still work!

Reply:
Ah thank you! Barge away 🛥